socorro/external/boto/crashstorage.py

```diff
@@ -218,17 +218,10 @@ class TelemetryBotoS3CrashStorage(BotoS3CrashStorage):
         from_string_converter=class_converter,
         reference_value_from='resource.boto'
     )
-    required_config.elasticsearch = Namespace()
-    required_config.elasticsearch.add_option(
-        'elasticsearch_class',
-        default='socorro.external.es.connection_context.ConnectionContext',
-        from_string_converter=class_converter,
-        reference_value_from='resource.elasticsearch',
-    )
```

Comment:
We don't need this anymore because `TelemetryS3CrashStorage` uses `SuperSearchFieldsData` which doesn't need a `ConnectionContext` or configuration.